.circleci/config.yml

```diff
@@ -284,12 +324,13 @@ workflows:
           name: build-<< matrix.target_os >>-<< matrix.target_arch >>
           matrix:
             parameters:
-              target_os: ["linux", "darwin"]
+              target_os: ["linux", "darwin", "windows"]
               target_arch: ["amd64", "arm64"]
               run_tests: [true]
             exclude:
-              - target_os: "linux"
+              - target_os: "windows"
                 target_arch: "arm64"
+                run_tests: true
```

Comment:
Previously this looks like it tried to exclude `linux-arm64` builds, but it didn't work (because all parameters need to be specified) and those builds have been running anyway. So I've removed that and only excluded `windows-arm64` builds – but we can put it back if it really was meant to be excluded.